setup.py

```diff
@@ -117,8 +120,8 @@ def make_packages(root_dir):
     license='Apache License 2.0',
     python_requires='>=3.7',
     install_requires=[
-        'th2-grpc-sim~=3.1.3',
-        'grpcio-tools==1.38.1'
+        'th2-grpc-sim~=5.1.0.dev',
+        'th2-grpc-common~=4.3.0.dev'
```

Comment:
Please, correct the versions. We should use the exact versions for Python to make sure that the right ones are used (there are no suffixes like in java. All dev versions look the same).
Please, also use `4.3.0.dev0` version for grpc-common. It is the latest dev release (we didn't use `rc*` at that moment)